pkgs/mo2installer/steam-redirector.nix

```diff
@@ -0,0 +1,43 @@
+{
+  pkgs,
+  overrideCC,
+  ...
+}: let
+  useWin32ThreadModel = stdenv:
+    overrideCC stdenv (
+      stdenv.cc.override (old: {
+        cc = old.cc.override {
+          threadsCross = {
+            model = "win32";
+            package = null;
+          };
+        };
+      })
+    );
+in
+  (useWin32ThreadModel pkgs.pkgsCross.mingwW64.stdenv).mkDerivation {
+    pname = "steam-redirector";
+    version = "5.0.3";
+
+    src = pkgs.fetchFromGitHub {
```

Comment:
`src` can be passed as an argument from the calling derivation, since it's the same.

Reply:
How would I do that?

Reply:
I'll push a commit, might be easier to show like that.